google_analytics/utils.py

```diff
@@ -76,6 +76,10 @@ def build_ga_params(
     else:
         client_ip = meta.get('REMOTE_ADDR', '')
 
+    # get user unique id
+    if hasattr(request.user, 'profile'):
+        uuid = request.user.profile.uuid
```

Comment:
Since this module is not used by us only, we'll need to add this to the params of `build_ga_params` like we do for title.